tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/NewApprenticeshipProvider/HidePassedNotificationTests.cs

```diff
@@ -0,0 +1,467 @@
+﻿using Dfc.CourseDirectory.WebV2.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Dfc.CourseDirectory.WebV2.Tests.FeatureTests.NewApprenticeshipProvider
+{
+    public class HidePassedNotificationTests : TestBase
+    {
+        public HidePassedNotificationTests(CourseDirectoryApplicationFactory factory)
+            : base(factory)
+        {
+        }
+
+
+        [Fact]
+        public async Task Post_FeOnlyProviderReturnsForbidden()
+        {
+            // Arrange
+            var providerId = await TestData.CreateProvider(apprenticeshipQAStatus: ApprenticeshipQAStatus.Passed, providerType: ProviderType.Apprenticeships);
+            await User.AsProviderUser(providerId, ProviderType.FE);
+
+            // Act
+            var response = await HttpClient.PostAsync(
+                $"new-apprenticeship-provider/hide-passed-notification?providerId={providerId}&returnUrl=/", null);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
+        }
+
+        [Fact]
+        public async Task Post_DeveloperUserReturnsBadRequest()
```

Comment:
Don't want this - Developer should be able to do everything

Reply:
removed.